src/werkzeug/middleware/profiler.py

```diff
@@ -44,11 +44,14 @@ class ProfilerMiddleware:
 
     -   ``{method}`` - The request method; GET, POST, etc.
     -   ``{path}`` - The request path or 'root' should one not exist.
-    -   ``{elapsed}`` - The elapsed time of the request.
+    -   ``{elapsed}`` - The elapsed time of the request in milliseconds.
     -   ``{time}`` - The time of the request.
 
-    If it is a callable, it will be called with the WSGI ``environ``
-    dict and should return a filename.
+    If it is a callable, it will be called with the following and should
+    return a filename:
+    -   ``{environ}`` - The WSGI ``environ`` dict.
+    -   ``{elapsed}`` - The elapsed time of the request in milliseconds.
```

Comment:
I wonder if we should put these in `environ["werkzeug.profiler"]["elapsed"]` etc. instead. That way the signature doesn't need to change, and more fields could be added later.

Reply:
Can do. I'd considered that, too, but opted for this so as not to pollute the WSGI dict.

Here are the options as I see them:

1) Add a `werkzeug.profiler` key with a dictionary value (your suggestion here)
    - pros: backwards and forwards compatible
    - cons: muddies the definition of the WSGI environ dict
2) Leave it as proposed in the PR
    - pros: doesn't muck with WSGI environ
    - cons: not backward or forward compatible
3) Leave it as is, but update docs to instruct clients to specify `**kwargs` in their function signature
    - pros: doesn't muck with WSGI environ, future compatible
    - cons: not backwards compatible, requires clients to future-proof by adding `**kwargs` to their function

Just let me know what you'd prefer, and I'll take care of it. Thanks for taking a look!

Reply:
I made the suggested changes to the code. If that looks good, I'll go ahead and update the docs to match.

Reply:
Docs are now updated.